lms/static/scripts/frontend_apps/test/errors-test.js

```diff
@@ -91,3 +93,79 @@ describe('isLTILaunchServerError', () => {
     });
   });
 });
+
+describe('formatErrorMessage', () => {
+  [
+    {
+      error: new Error('This is any old error'),
+      expected: 'This is any old error',
+    },
+    {
+      error: new APIError(400, { message: 'This is a server error' }),
+      expected: 'This is a server error',
+    },
+    {
+      error: new APIError(400, {}),
+      expected: '',
+    },
+    {
+      error: new APIError(400, {}),
+      prefix: 'Something went wrong',
+      expected: 'Something went wrong',
+    },
+    {
+      error: new APIError(404, { message: 'This is a server explanation' }),
+      prefix: 'Something went wrong',
+      expected: 'Something went wrong: This is a server explanation',
+    },
+    {
+      error: new Error('Any old error'),
+      prefix: 'Something went wrong',
+      expected: 'Something went wrong: Any old error',
+    },
+  ].forEach((testCase, idx) => {
+    it(`should format the error message (${idx})`, () => {
+      assert.equal(
+        formatErrorMessage(testCase.error, testCase.prefix),
+        testCase.expected
+      );
+    });
+  });
+});
+
+describe('formatErrorDetails', () => {
+  [
+    {
+      error: { details: { foo: 'bar' } },
+      expected: JSON.stringify({ foo: 'bar' }, null, /** indent */ 2),
```

Comment:
```suggestion
      expected: JSON.stringify({ foo: 'bar' }, null, /* indent */ 2),
```

This is not a JSDoc comment.